### DISCUSSION TITLE
Is there a simple way to convert a JVP into a VJP?

### DISCUSSION THREAD

**Discussion by NeilGirdhar** [jax-ml/jax#18635] [General] [answered]
Flax has a lifted VJP, but no lifted JVP.  I wonder if there's an easy way to convert one to the other?

**Comment by patrick-kidger** [comment, marked as answer]
`jax.linear_transpose`?

**Reply by NeilGirdhar** [reply]
Ah yes, thanks!  I knew I had read somewhere that it was possible.  Unfortunately, Flax doens't implement `linear_transpose` either.
